front/src/components/mmt-accommodation-card/mmt-accommodation-card.component.js

```diff
@@ -0,0 +1,30 @@
+import {
+  VCard, VCardTitle, VCardSubtitle, VCardText,
+  VCarousel, VCarouselItem, VImg,
+  VRating, VIcon
+} from 'vuetify/lib';
+const { baseUrl } = require('@/config/config.dev.json');
+const { SERVER_ACCOMODATIONS_PATH } = require('@constants');
```

Comment:
i think it could be managed in server-database level - why we can't save in db with folder path?